components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/test/java/org/wso2/carbon/identity/role/v2/mgt/core/dao/RoleDAOTest.java

```diff
@@ -520,6 +549,72 @@ public void testUpdateRoleName() throws Exception {
 
         assertTrue(roleDAO.isExistingRoleName("newRole", APPLICATION_AUD, SAMPLE_APP_ID,
                 SAMPLE_TENANT_DOMAIN));
+
+        // Constructing a shared role scenario
+        RoleBasicInfo roleBasicInfo = addRole("sharing-org-role-001", ORGANIZATION_AUD, SAMPLE_ORG_ID, roleDAO);
```

Comment:
Let's break this to a different test method

There are lot of samples available inside this test.

The initial part is about updating an application audience role name. Newly added part if about renaming an org audiene role.
Better to have two different tests